scripts/src/main/java/io/github/fate_grand_automata/scripts/entrypoints/AutoServantLevel.kt

```diff
@@ -250,9 +267,43 @@ class AutoServantLevel @Inject constructor(
      * @see isAutoSelectVisible
      */
     private fun performAutoSelect() {
+        if (stateAutoFill) {
+            locations.enhancementClick.click()
+            0.5.seconds.wait()
+            val exist = locations.servant.finalConfirmRegion.exists(
+                image = images[Images.Ok],
+                timeout = 3.seconds,
```

Comment:
Which situation are you checking here that isn't checked by `isEmptyEmberOrQPDialogVisible()`?

Reply:
When you ran out of embers, it doesn't show any dialog. This will let the app knows it has no more embers